lms/services/h_api.py

```diff
@@ -42,21 +42,22 @@ def get_user(self, username):
             display_name=user_info["display_name"],
         )
 
-    def create_user(self, h_user, provider, provider_unique_id):
+    def create_user(self, h_user):
         """
         Create a user in H.
 
         :arg h_user: the user to be created in h
         :type h_user: HUser
-        :param provider: the "provider" string to send to h
-        :param provider_unique_id:  the "provider_unique_id" string to send to h
         """
         user_data = {
             "username": h_user.username,
             "display_name": h_user.display_name,
             "authority": self._authority,
```

Comment:
```suggestion
            "authority": h_user.authority,
```

I think this works, and it would make it clearer that this is all about the user. Might pave the way for removing `_authority` from this object in future if it's not used.

Reply:
Done. We're unlikely to be able to remove `authority` from `HAPIService` though, it's used for a bunch of things